build-data-capturing-gradle-samples/capture-dependency-resolution/gradle-dependency-resolution.gradle

```diff
@@ -8,22 +7,24 @@
 
 def buildScanApi = project.extensions.findByName('buildScan')
 if (!buildScanApi) {
-    return
+  return
 }
 
+//FIXME Make Configuration cache compatible
```

Comment:
i would add a note in the description of this script that it is not configuration cache compatible.